manifests/package.pp

```diff
@@ -182,7 +182,7 @@
 
     exec { 'remove_plugin_dir':
       refreshonly => true,
-      command     => "rm -rf ${elasticsearch::_plugindir}",
+      command     => "rm -rf ${::elasticsearch::_plugindir}",
```

Comment:
Wasn't this a workaround for puppet3 and variable scoping? do we still need the `::` prefix?

Reply:
This was to resolve the failing unit tests. It appears to be the only variable in that manifest missing  the colons.

Reply:
hmm, ok :)